src/tests/ftest/util/soak_utils.py

```diff
@@ -1209,7 +1213,7 @@ def create_app_cmdline(self, job_spec, pool, ppn, nodesperjob):
             if not self.enable_il and api in ["POSIX-LIBIOIL", "POSIX-LIBPIL4DFS"]:
                 continue
             add_containers(self, pool, file_oclass, dir_oclass)
-            sbatch_cmds = ["module purge", f"module load {self.mpi_module}"]
+            sbatch_cmds = [f"module use {self.mpi_module_use}", f"module load {self.mpi_module}"]
```

Comment:
... here, could we do?
```
sbatch_cmds.append(f"module use {mpi_module_use};module load {mpi_module}")
```

Reply:
need to verify if dfuse can run with mpi_module*  (i.e not self.mpi_module*)